shared/src/main/kotlin/org/javacs/kt/Logger.kt

```diff
@@ -43,10 +43,17 @@ enum class LogLevel(val value: Int) {
 
 class LogMessage(
     val level: LogLevel,
-    val message: String
+    val message: String,
+    private val funName: String? = null,
 ) {
     val formatted: String
-        get() = "[$level] $message"
+        get() {
+            return if (funName != null) {
+                "[$level] $funName $message"
+            } else {
+                "[$level] $message"
+            }
+        }
```

Comment:
```suggestion
        get() = "[$level] ${funName?.let { "$it " } ?: ""}$message"
```

Maybe there's a more elegant way